sormas-app/app/src/main/res/layout/fragment_event_edit_layout.xml

```diff
@@ -240,29 +240,71 @@
 
             </LinearLayout>
 
+
+
             <de.symeda.sormas.app.component.controls.ControlTextPopupField
                 android:id="@+id/event_eventLocation"
                 style="@style/ControlSingleColumnStyle"
                 app:hint="@string/hint_enter_location"
                 app:iconStart="@drawable/ic_edit_location_black_24dp"
                 app:locationValue="@={data.eventLocation}" />
 
+
+            <LinearLayout
+                android:layout_width="match_parent"
+                android:layout_height="wrap_content"
+                android:orientation="horizontal">
+
             <de.symeda.sormas.app.component.controls.ControlSpinnerField
                 android:id="@+id/event_meansOfTransport"
-                style="@style/ControlSingleColumnStyle"
+                style="@style/ControlFirstOfTwoColumnsStyle"
                 app:dependencyParentField="@{eventTypeOfPlace}"
                 app:dependencyParentValue="@{TypeOfPlace.MEANS_OF_TRANSPORT}"
                 app:softRequired="true"
                 app:value="@={data.meansOfTransport}" />
 
+
             <de.symeda.sormas.app.component.controls.ControlTextEditField
```

Comment:
Details should have an individual row, half of a row is not enough space for a free text field